config/initializers/db_warehouse.rb

```diff
@@ -1,4 +0,0 @@
-# Rails.logger.debug "Running initializer in #{__FILE__}"
-
-# save health database settings in global var
-DB_WAREHOUSE ||= YAML.safe_load(ERB.new(File.read(Rails.root.join("config","database.yml"))).result, aliases: true)[Rails.env]['warehouse']
```

Comment:
We seemed to be using this to with establish_connection which is probably not correct (transactions aren't shared for one). Instead should use `connects_to [database]`